### PR TITLE
Add missing resource

### PR DESCRIPTION
**Pull request by Forgind**
### Context
@dfederm tried to see what the error was when he specified /low:Foo and got...a missing resource exception. Whoops. This fixes that.

### CODE REVIEW

**Review by dfederm — APPROVED** [review]

File: src/MSBuild/Resources/Strings.resx [hunk 1]
```diff
@@ -1235,6 +1235,15 @@
       LOCALIZATION: The prefix "MSBUILD : error MSBxxxx:" should not be localized.
     </comment>
   </data>
+  <data name="InvalidLowPriorityValue" UESanitized="true" Visibility="Public">
+    <value>MSBUILD : error MSB1064: Low priority value is not valid. {0}</value>
```
Comment (JanKrivanek) [inline src/MSBuild/Resources/Strings.resx:1239]: Is the MSB code worth documenting anywhere?

**Review by JanKrivanek — APPROVED** [review]

File: src/MSBuild/Resources/Strings.resx (hunk 1 above)
Comment (Forgind) [reply src/MSBuild/Resources/Strings.resx:1239]: I'd ask @ghogen, but my personal opinion is "only if we have a list of error codes for invalid switches already." It took a long time for anyone to even see this, and when someone did, they had intentionally introduced an error. So documentation for this error is "low priority" 😉

**Review by rainersigwald — APPROVED** [review]
Please update https://github.com/dotnet/msbuild/blob/66eccea0589072be7303f567377fa6f6c479adab/src/MSBuild/Resources/Strings.resx#L1489

**Comment by Forgind** [comment]
> Please update
> 
> https://github.com/dotnet/msbuild/blob/66eccea0589072be7303f567377fa6f6c479adab/src/MSBuild/Resources/Strings.resx#L1489

As an aside, I skipped MSB1063 because that's currently in use in my other PR.

**Comment by danmoseley** [comment]
Can resources lookups be made strongly typed so this kind of bug is not possible?

**Pull request merged**